spec/api-net-spec.ts

```diff
@@ -1192,7 +1192,7 @@ describe('net module', () => {
           expect.fail('Request should not be intercepted by the default session');
         });
 
-        const customSession = session.fromPartition(customPartitionName, { cache: false });
+        const customSession = session.fromPartition(customPartitionName, { cache: false, quota: 0 });
```

Comment:
What would happen if quota isn't specified? Existing functionality should work as it is without any modifications after our change. If this quota parameter is mandatory, we should reconsider the design.

Reply:
In a normal case this works, a test case I added using browser window is fine with just having the cache option enabled. Hwoever, since the test is in TS, I am unsure if there are special rules which apply.

Reply:
We should leave existing test cases as they are. There should be no change needed in existing code to work as it was working before.